src/calculateRentalCost.js

```diff
@@ -28,7 +28,24 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let result;
+  const rentCost = days * 40;
+  const cashBack3MoreDays = 20;
+  const cashBack7MoreDays = 50;
+
+  if (days < 3) {
+    return rentCost;
+  }
+
+  if (days >= 3 & days < 7) {
+    result = rentCost - cashBack3MoreDays;
+  }
```

Comment:
```suggestion
  if (days < 7) {
    result = rentCost - cashBack3MoreDays;
  }
```
you returned value from function in previous if, so no need to check it here